apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -313,7 +319,10 @@ class TrygdetidServiceImpl(
                     ident = avdoedMedFnr.first,
                     yrkesskade = false,
                 )
-            val opprettetTrygdetid = trygdetidRepository.opprettTrygdetid(trygdetid)
+            val pesystt = pesysKlient.hentTrygdetidsgrunnlag(avdoedMedFnr.first, brukerTokenInfo)
+            val oppdatertTrygdetidv2 = populertrygdetidFraPesys(trygdetid, pesystt)
```

Comment:
hva med `trygdetidMedPerioderFraPesys`. Litt tydeligere hva som ligger her

Reply:
Fjerner det her og legger til egen knapp for å sjekke